actionpack/lib/action_dispatch/system_test_case.rb

```diff
@@ -20,7 +20,7 @@ class SystemTestCase < IntegrationTest
     # <tt>application_system_test_case.rb</tt> file that is generated with a new
     # application or scaffold.
     #
-    # Here is an example system test:
+    # Here is an example of system test:
```

Comment:
The current one is ok. If this needs to change it should be- "example of a system test".